channeldb/invoices.go

```diff
@@ -1893,17 +1893,20 @@ func (d *DB) updateInvoice(hash *lntypes.Hash, refSetID *invpkg.SetID, invoices,
 		return &invoice, nil
 	}
 
+	switch update.UpdateType {
+	case invpkg.CancelHTLCsUpdate:
+		return d.cancelHTLCs(invoices, invoiceNum, &invoice, update)
+	}
+
 	var (
-		newState = invoice.State
-		setID    *[32]byte
+		setID *[32]byte
 	)
 
 	// We can either get the set ID from the main state update (if the
 	// state is changing), or via the hint passed in returned by the update
 	// call back.
 	if update.State != nil {
 		setID = update.State.SetID
-		newState = update.State.NewState
```

Comment:
nit: this seems to be the wrong commit to remove it, was it because it was previously used in `cancelSingleHtlc` and now the compiler is complaining if it's not removed?